modules/ROOT/pages/release-notes.adoc

```diff
@@ -139,6 +139,21 @@ bonita.tenant.connector.warnWhenLongerThanMillis
 
 More information xref:performance-tuning.adoc[here]
 
+===== BDM
```

Comment:
Be carefull, title lvl 5 doesn't render really well on the doc site. 
It's not recommended to have so many levels levels on a documentation page, it impacts readability if everything is a title (that's why title lvl 5 are not highlighted). 
It would be better to re work the release note structure to avoid all this hierarchy. 

(I know it's not the purpose of this contribution and not really your responsibility, but if someone like @pabloalonso or @NathalieC could have a look :) )

Reply:
done